main/src/ca/uwaterloo/flix/api/lsp/provider/HoverProvider.scala

```diff
@@ -63,7 +63,11 @@ object HoverProvider {
   }
 
   private def hoverTypAndEff(tpe: Type, eff: Type, loc: SourceLocation)(implicit index: Index, root: Root): JObject = {
-    val markup = formatTypAndEff(tpe, eff)
+    val markup =
+      s"""```flix
+         |${formatTypAndEff(tpe, eff)}
+         |```
+         |""".stripMargin
```

Comment:
Does this extra line break cause an extra line break to show up or is it not a problem?

Reply:
I haven't noticed any superfluous line breaks showing up in VS Code. Here's an image showing the hover for local variables (which are handled by that method). 🙂 

![flix-hover-1](https://user-images.githubusercontent.com/129259/132092066-81850e91-6822-4906-88f9-50eb328e0833.png)